src/item/Bow.php

```diff
@@ -46,7 +46,13 @@ public function getMaxDurability() : int{
 
 	public function onReleaseUsing(Player $player) : ItemUseResult{
 		$arrow = VanillaItems::ARROW();
-		if($player->hasFiniteResources() and !$player->getInventory()->contains($arrow)){
+		$inventory = match(true){
+			$player->getOffHandInventory()->contains($arrow) => $player->getOffHandInventory(),
+			$player->getInventory()->contains($arrow) => $player->getInventory(),
+			default => null
+		};
+
+		if($player->hasFiniteResources() and $inventory === null){// Receives a Mismatch Transaction when using offhand arrows. why???
```

Comment:
This comment can be removed. It's not something that can be fixed by this part of the code anyway.